scripts/cypress.js

```diff
@@ -3,27 +3,35 @@ const cypress = require('cypress');
 const path = require('path');
 
 /**
- * Script that run/opens cypress, since cypress does not support easy config extension
- * Can be removed in favour of native CLI once cypress supports path based config extension
+ * Script that run/opens cypress, since cypress does not support easy config extension.
+ * Can be removed in favour of native CLI once cypress supports path-based config extension.
  * https://github.com/cypress-io/cypress/issues/5674
```

Comment:
cypress has done some reorganizing of their issues, can you please update to the recent one ?
```suggestion
 * https://github.com/cypress-io/cypress/issues/5218
```